chef/cookbooks/crowbar/templates/default/crowbar-lbaas-evacuation.sh.erb

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# This script migrates all neutron lbaasv2 loadbalancers from the node
+# that is passed in via the command line argument.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+UPGRADEDIR=/var/lib/crowbar/upgrade
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+hostname=$1
+if [[ "$2" = "delete-ns" ]]; then
+    delete_ns="--delete_namespaces --nosource_agent_restart"
+else
+    delete_ns=""
+fi
+
+log()
+{
+    set +x
+    echo "[$(date --iso-8601=ns)] [$$] $@"
+    set -x
+}
+
+log "Executing $BASH_SOURCE"
+
+set -x
+
+mkdir -p $UPGRADEDIR
+rm -f $UPGRADEDIR/crowbar-lbaas-evacuation-failed
+
+systemctl --quiet is-active  openstack-neutron-lbaasv2-agent.service
+ret=$?
+if [ $ret = 0 ] ; then
+    use_crm=<%= @use_ha ? "--use_crm" : "" %>
+    /usr/bin/neutron-evacuate-lbaasv2-agent $use_crm $delete_ns \
```

Comment:
Maybe it makes sense to `log` something (=timestamp) before execution of the command.